README.md

```diff
@@ -156,7 +156,7 @@ _this is the recommended way to run remark42_
 | auth.email.subj         | AUTH_EMAIL_SUBJ         | `remark42 confirmation`  | email subject                                   |
 | auth.email.content-type | AUTH_EMAIL_CONTENT_TYPE | `text/html`              | email content type                              |
 | auth.email.template     | AUTH_EMAIL_TEMPLATE     | none (predefined)        | custom email message template file              |
-| notify.users            | NOTIFY_USERS            | none                     | type of user notifications (email)              |
+| notify.users            | NOTIFY_USERS            | none                     | type of user notifications (telegram, email)    |
```

Comment:
probably we need some detailed info about user's tg notif somewhere

Reply:
Please elaborate on this one, documentation note on what is the trickiness of telegram notify?